python/tests/datasources/test_acs_condition.py

```diff
@@ -37,7 +37,7 @@ def _get_by_race_as_df(*args):
 @mock.patch('ingestion.gcs_to_bq_util.load_values_as_df',
             side_effect=_get_by_race_as_df)
 def testSexNational(mock_acs: mock.MagicMock, mock_fips: mock.MagicMock):
-    acsHealthInsurance = AcsHealthInsurance()
+    acsHealthInsurance = AcsCondition()
```

Comment:
should probably rename the instantiated class variable to something more generic since we aren't just testing health insurance right? something like `datasource = AcsCondition()` or whatever

Reply:
Yes